src/test/regress/expected/multi_extension.out

```diff
@@ -551,11 +551,12 @@ SELECT * FROM print_extension_changes();
 -- Test downgrade to 10.0-3 from 10.1-1
 ALTER EXTENSION citus UPDATE TO '10.1-1';
 ALTER EXTENSION citus UPDATE TO '10.0-3';
--- Should be empty result since upgrade+downgrade should be a no-op
 SELECT * FROM print_extension_changes();
- previous_object | current_object
+ previous_object |                    current_object
 ---------------------------------------------------------------------
-(0 rows)
+                 | function columnar.columnar_metapage(regclass)
+                 | function columnar.upgrade_columnar_metapage(regclass)
+(2 rows)
```

Comment:
Done, though we still have a "citus_internal.upgrade_columnar_storage()"

Reply:
Since we need to drop `citus_internal.upgrade_columnar_storage` in `columnar--10.1-1--10.0-3.sql`, we shouldn't have that here.
Instead, we would have `citus_internal.upgrade_columnar_storage` around `multi_extension.out::L-563`

Reply:
I merged your suggested commit. Thank you!